app/api/search.v2/routes.ts

```diff
@@ -0,0 +1,63 @@
+import { Application, Request, Response, NextFunction } from 'express';
+//@ts-ignore
+import boolParser from 'express-query-boolean';
```

Comment:
let me check.

Reply:
since im removing query-string in favor of qs im going remove boolParser and use the recommended way from qs https://github.com/ljharb/qs#parsing-primitivescalar-values-numbers-booleans-null-etc, it also relies on a library that seems not maintained, the  source code is really small, if we have any problem with this we can take care of this ourselves.